src/main/java/Wolox/training/models/Book.java

```diff
@@ -122,6 +130,7 @@ public String getYear() {
     }
 
     public void setYear(String year) {
+        checkNotNull(year);
```

Comment:
As I mentioned before about checking for empty strings, here you must also check that it is a numeric value